content/_getting-started/vpn-agent.md

```diff
@@ -0,0 +1,92 @@
+---
+title: VPN Agent
+layout: article
+section: Introduction
+description: This document provides the basic information on VPN Agents and the way setup one in case you require one.
+order: 7
+since: 20200730
+category: intro
+---
+
+This document provides basic information on [VPN Agent](#vpn-agent), [how to set it up](#how-to-setup)
+and [use it in your integration flow](#using-vpn-agent).
+
+## VPN Agent
+
+The VPN Agent or the Agent is а VPN tunnel specifically configured to access any
+local resource in your local environment. It could be any local database or CRM
+to which you would rather restrict access from outside world but still need
+to query for a data.
+
+When the Agent is configured the platform and your local resource communicate data
+using the secure VPN tunnel. All the processing is happening on platform side, your
+resource serves the role of data source or final destination of data depending on
+your use case.
+
+## How to setup
+
+To setup the Agent you would need to fulfill the following preconditions:
+
+*   **You have a running local resource which has a local IP address and port.** The IP address should not be `127.0.0.1` or instead be `localhost`. Please consult the documentation of your resource on how to configure it.
+*   **Working VPN client which accepts OpenVPN configurations.**
+*   Optionally: Administrative rights to configure the local network on your host machine in case you have non-standard setup.
+
+### Connecting to local database
+
+As an example let us connect the locally running Mongo database with the platform.
+Let's assume our local database is accessible on `192.168.1.7` address and `27017`
+port (standard MongoDB port).
+
+Navigate to left-hand-side menu, Integrate > Agents and press on **Create an Agent** button
+if you don't have any Agents or press on **Create New Agent** button to add new one.
+
+You will get a new pop-up form to fill in the details. Note you can extend the form
+by selecting **Create New Endpoint**.
+
+{% include img.html max-width="60%" url="/assets/img/getting-started/vpn-agent/vpn-agent-form-empty.png" title="VPN Agent request form" %}
+
+Let us fill in the needed values:
+
+1.  Give a proper name to remember what is it for. We will use `MongoDB_local_com`.
+2.  Select from drop-down menu the connection protocol type `TCP` or `UDP`. We use `TCP` becuase of our configured database interface has TCP protocol.
```

Comment:
because we use mongodb, and it uses tcp protocol

Reply:
Done, thanks for correction.